R/presto.field_utilities.R

```diff
@@ -0,0 +1,166 @@
+print.presto.field <- function(prf, prefix = "", ...) {
```

Comment:
add copyright header

Reply:
Done. Also added in the presto.fields.R file.